lagscope/lagscope.Config.json

```diff
@@ -21,13 +21,5 @@
         "Time"           : 60,
         "PingIterations" : 20000,
         "Options"        : ""
-    },
-
-    "LagscopeTest": {
-        "Iterations"     : 0,

```

Comment:
you mean something greater than 0?

Reply:
I think this is removing a testconfig that I had created and merged on accident.